libraries/ESP8266HTTPClient/examples/Authorization/Authorization.ino

```diff
@@ -40,22 +42,24 @@ void loop() {
   // wait for WiFi connection
   if ((WiFiMulti.run() == WL_CONNECTED)) {
 
+    WiFiClient client;
+
     HTTPClient http;
 
     USE_SERIAL.print("[HTTP] begin...\n");
     // configure traged server and url
 
 
-    http.begin("http://user:password@192.168.1.12/test.html");
+    http.begin((Client&)client, "http://guest:guest@jigsaw.w3.org/HTTP/Basic/");
```

Comment:
Isn't there no need for the cast `(Client&)` here and in the other examples?  Your prototype will make the C++ compiler pass the pointer no matter what, no?

Reply:
You will be right. I'll check it and then change it

Reply:
The cast has been removed